data/styles/clear/include/Icons.mapcss

```diff
@@ -2010,6 +2010,7 @@ node|z16-[barrier=stile],
 {icon-image: dot-m.svg;font-size: 11;}
 
 node|z16-[highway=ford],
+/* ToDo: Make separate tower icon? Because of a good landmark */
```

Comment:
But its quite a lot of them also (where they're mapped)...
In some cartographic standards power towers/poles are shown as small circles, so the dot is OK fit also.
(btw, I've tried to use `symbol-shape: circle`, but it looks this drule type doesn't work... or maybe I did something wrong)

Reply:
What's in the code for it?